internal/tiltfile/links/links.go

```diff
@@ -117,6 +101,10 @@ func (e Extension) link(thread *starlark.Thread, fn *starlark.Builtin, args star
 	}
 
 	return Link{
+		Struct: starlarkstruct.FromStringDict(starlarkstruct.Default, starlark.StringDict{
+			"url":  starlark.String(withScheme),
+			"name": starlark.String(name),
+		}),
 		Link: model.Link{
```

Comment:
It feels a bit awkward to have this data in here twice - maybe it'd be less awkward to remove this field and have `LinkList.Unpack` convert from `links.Link` to `model.Link`?

Reply:
i played around with this and the conversion logic from a generic Struct to a model.Link would be complicated/error-prone (i.e., the struct interface isn't really made for this kind of conversion), so i'm not sure that'd be less awkward